webapp/src/ts/app.component.ts

```diff
@@ -643,4 +651,20 @@ export class AppComponent implements OnInit {
     // avoid Failed to fetch errors being logged when the browser window is reloaded
     this.changesService.killWatchers();
   }
+
+  private async initAnalyticsModules() {
+    try {
+      const canAccess= await this.headerTabsService.canAccessTab('analytics');
+
+      if (!canAccess) {
+        return;
+      }
+
+      const modules = await this.analyticsModulesService.get();
+      this.analyticsActions.setAnalyticsModules(modules || []);
+
+    } catch (error) {
+      console.error(error);
```

Comment:
Please add some additional text here to signify where this error is coming from: 
```suggestion
      console.error('Error while initializing analytics modules', error);
```

Reply:
Done